packages/types/src/api-reference/html-rendering-configuration.ts

```diff
@@ -0,0 +1,29 @@
+import type { ApiReferenceConfiguration } from '@/api-reference/api-reference-configuration.ts'
+import { z } from 'zod'
+
+/**
+ * Zod schema for HTML rendering configuration
+ */
+export const htmlRenderingConfigurationSchema = z.object({
+  /**
+   * The URL to the Scalar API Reference JS CDN.
+   *
+   * Use this to pin a specific version of the Scalar API Reference.
+   *
+   * @default https://cdn.jsdelivr.net/npm/@scalar/api-reference
+   *
+   * @example https://cdn.jsdelivr.net/npm/@scalar/api-reference@1.25.122
+   */
+  cdn: z.string().optional().default('https://cdn.jsdelivr.net/npm/@scalar/api-reference'),
+  /**
+   * The title of the page.
+   */
+  pageTitle: z.string().optional().default('Scalar API Reference'),
+})
+
+/**
+ * The configuration for the static HTML rendering using the CDN.
+ *
+ * It’s the ApiReferenceConfiguration, but extended with the `pageTitle` and `cdn` options.
+ */
+export type HtmlRenderingConfiguration = ApiReferenceConfiguration & z.infer<typeof htmlRenderingConfigurationSchema>
```

Comment:
```suggestion
export type HtmlRenderingConfiguration = Partial<ApiReferenceConfiguration> & z.infer<typeof htmlRenderingConfigurationSchema>
```
that should be a partial I believe as its for user input